packages/database/src/core/view/ChildChangeAccumulator.ts

```diff
@@ -89,14 +88,11 @@ export class ChildChangeAccumulator {
         );
       }
     } else {
-      this.changeMap_[childKey] = change;
+      this.changeMap.set(childKey, change);
     }
   }
 
-  /**
-   * @return {!Array.<!Change>}
-   */
   getChanges(): Change[] {
-    return getValues(this.changeMap_);
+    return Array.from(this.changeMap.values());
```

Comment:
This is not supported in IE11. Do you think the changes in this file warrant another Polyfill requirement?

Reply:
Probably. Man, that file's going to be really hard to keep in sync if we add every feature one by one. :confused:

Reply:
Added to polyfill package.